samplesheets/assayapps/generic/plugins.py

```diff
@@ -96,7 +100,13 @@ def _get_col_value(cls, target_col, row, table):
                 header = table['field_header'][i]
                 if header['value'].lower() == target_col.lower():
                     value = row[i]['value']
-        return value
+
+        if isinstance(value, str):
+            return value
+        elif isinstance(value, list) and len(value) == 1:  # OntologyTermRefs
+            return value[0]['name']
+        else:
```

Comment:
Here we can leave out the `else:` and simply `return None`, as reaching any of the previous clauses returns from the function anyway.